module2/projects/coupon-codes-api/evaluation.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Coupon Codes API Evaluation
+---
+
+## Evaluation Criteria
+
+_[Back to Coupon Codes Home](./index)_
+
+## Presentation
+For the project evaluation, you will meet with an instructor for a 20 minute session. You should prepare a presentation that covers the following:
+
+<!-- TO-DO Change to using FE to demonstrate functionality? -->
+* Demonstration of functionality via Postman suites
+* Discussion of technical quality and organization of the code, identifying code that should be refactored and how it would be refactored
+* Running your application's test suite and a discussion of test coverage (happy/sad paths and any edge cases)
+* Identifying the area(s) of code of which you are most proud, and an area where you would like specific feedback
+
+Students should focus on practicing technical communication that is succinct and utilizes appropriate technical vocabulary.
+
+Slides are not required, but encouraged as a way to facilitate the presentation along with sharing specific code examples.
+
+With remaining time, instructors will ask questions and follow-up on feedback requests.
+
+
+## Rubric
+Your project will be evaluated based on the following rubric: 
+
+### Feature Delivery
+
+* Exceeds Expectations: Project completes all requirements and at least one extension.
+* Meets Expectations: Project completes all requirements. Postman tests pass, and FE functions correctly when wired up to API. 
+* Approaching Expectations: Project fails to complete 1 - 2 required endpoints
+* Below Expectations: Project fails to complete more than 3 or more endpoints
```

Comment:
Suggestion: 
* Below Expectations: Project fails to complete 3 or more endpoints